pkg/assembler/backends/inmem/hasSBOM.go

```diff
@@ -273,6 +362,8 @@ func (c *demoClient) addHasSBOMIfMatch(out []*model.HasSbom,
 	filter *model.HasSBOMSpec, link *hasSBOMStruct) (
 	[]*model.HasSbom, error) {
 
+	// TODO(knrc) - Should we include the included software, dependencies and occurrences?
+	// Need a better definition of what is required for this.
```

Comment:
we could do something similar to `hasSLSA`, which also contains an array of artifacts (`builtFrom`): https://github.com/guacsec/guac/blob/main/pkg/assembler/backends/inmem/hasSLSA.go#L328

Reply:
Thanks, will take a look

Reply:
This is now included